lib/hyperclient.rb

```diff
@@ -7,16 +7,23 @@
 require 'hyperclient/resource'
 require 'hyperclient/resource_collection'
 require 'hyperclient/version'
+require 'hyperclient/railtie' if defined?(Rails)
 
 # Public: Hyperclient namespace.
 #
 module Hyperclient
+  URL_TO_ENDPOINT_MAPPING = { }
```

Comment:
This isn't a constant, shouldn't be `CAPS`.

Reply:
This reference seems constant, as it's the object itself that will be mutated, right?